sig-cluster-lifecycle/charter.md

```diff
@@ -34,7 +32,8 @@ The following topics fall under ownership of this SIG:
 - User interface, or user experience, issues other than cluster bootstrapping or management (see [sig-ui](../sig-ui) and  [sig-cli](../sig-cli)).
 - Node related issues (see [sig-node](../sig-node)).
 - Kubernetes control plane issues:
-   - Control plane component related issues (see [sig-api-machinery](../sig-api-machinery)).
+   - Control plane component related issues (see [sig-api-machinery](../sig-api-machinery) and [sig-scheduling](../sig-scheduling)).
```

Comment:
[3]

Reply:
does this imply that sig-api-machinery owns https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-component-upgrade-order

Reply:
> Control plane component related issues

refers to bugs around kube-apiserver and kube-controller-manager owned by SIG API Machinery and kube-scheduler owned by SIG Scheduling.

> does this imply that sig-api-machinery owns

i'd assume that given the upgrade order guideline is cutting across multiple core components that the real owner of the should be SIG Architecture with technical guidance by the component owning SIGs.

> Tools that assist in the creation, configuration, upgrade, downgrade, and teardown of Kubernetes control plane components.

with the order in place tools created by SIG Cluster Lifecycle should follow the guideline, as much as possible, or at least this is my interpretation.